packages/dds/merge-tree/src/index.ts

```diff
@@ -95,7 +95,12 @@ export {
 	SegmentGroup,
 	toRemovalInfo,
 } from "./mergeTreeNodes";
-export { Trackable, TrackingGroup, TrackingGroupCollection } from "./mergeTreeTracking";
+export {
+	Trackable,
+	TrackingGroup,
+	UnorderedTrackingGroup,
```

Comment:
is it necessary to expose? i would have hoped we could expose the interface but not the class